MidwestAccessCoalition/lockbox_rails
# Fix sidekiq in prod

## Changelog
- reworks note creation such that it's created as a background job
- adds sidekiq.yml and adds worker startup to the Procfile
- adds database cleaner gem (to address test failures from leftover test data)
- ensures RAILS_ENV is set before test dependencies are loaded

## Link to issue:  
Fixes issue https://github.com/MidwestAccessCoalition/lockbox_rails/issues/403


## Steps for QA/Special Notes:
- Changes have been deployed to demo-lockbox -- file a support request and check for note creation to ensure that notes were created (via sidekiq worker)
- When signed in as an admin user, you can view the sidekiq dashboard at /sidekiq
- Note -- post deploy, one must manually scale up the number of dynos via the heroku CLI so that there are dynos available to drain the queue

## Relevant Screenshots: 
- super important for UI tasks!


## Are you ready for review?:
